AutoDI/ContainerMap.cs

```diff
@@ -108,6 +108,8 @@ public object Get(Type key, IServiceProvider provider)
                         .Invoke(this, new object[] { provider });
                 }
             }
+            //Type key not found
+            RaiseTypeKeyNotFoundEvent(new TypeKeyNotFoundEventArgs(key));
```

Comment:
Since this event is only even going to be raised internally by this class I think we should ditch the raise method and just do it directly.
```C#
TypeKeyNotFoundEvent?.Invoke(this, new TypeKeyNotFoundEventArgs(key));
```

Reply:
Done.